workspaces/github-actions/plugins/github-actions/README.md

```diff
@@ -100,7 +100,6 @@ yarn --cwd packages/app add @backstage-community/plugin-github-actions
 // In packages/app/src/components/catalog/EntityPage.tsx
 import {
   EntityGithubActionsContent,
-  isGithubActionsAvailable,
```

Comment:
I think it would be better to show how it's used, ideally it would be part of the `if` prop on line 111. I think this might have been removed over the last few larger changes made to this file.